cli/cmd/auth.go

```diff
@@ -148,6 +148,13 @@ func init() {
 }
 
 func verifyAuthParams(authParams *authCmdParams) error {
+
+	const parametersRequiredMessage = "keptn auth requires api-token and endpoint \n\n" +
+		"For more information on how to obtain token and endpoint to go https://keptn.sh/docs/%s/reference/cli/#authenticate-keptn-cli \n\n" +
+		"Alternatively,To quickly access Keptn, you can use a port-forward and then authenticate your Keptn CLI: \n" +
```

Comment:
I think there is a space missing between *Alternatively,* and *To*. Also, *To* should be lowercased *to*

Reply:
Fixed in 42c037f666185b728f27f042d7e3faca704c5ec5 !